ui/src/app/navigation/commoncontrols.component.ts

```diff
@@ -14,10 +17,24 @@ export class CommoncontrolsComponent extends AbstractSearchComponent implements
   searchIcon = SvgHelper.path(SvgIcon.SEARCH)
   dismissIcon = SvgHelper.path(SvgIcon.DISMISS)
 
-  constructor(protected searchService: SearchService, protected route: ActivatedRoute, protected authService: AuthService) {
+  constructor(
```

Comment:
This is legit, but weird. Please annotate why we put it here with a code comment.